pythonosc/osc_message_builder.py

```diff
@@ -56,25 +69,44 @@ def add_arg(self, arg_value, arg_type=None):
     Raises:
       - ValueError: if the type is not supported.
     """
-    if arg_type and arg_type not in self._SUPPORTED_ARG_TYPES:
+    if arg_type and not self.valid_type(arg_type):
       raise ValueError(
           'arg_type must be one of {}'.format(self._SUPPORTED_ARG_TYPES))
```

Comment:
this error message also need to be fixed now that you're checking for another error case